### DISCUSSION TITLE
Getting started with an Image Editor in Reflex.dev?

### DISCUSSION THREAD

**Discussion by rjachuthan** [reflex-dev/reflex#2658] [Ideas]
I have to create a web app for my work and I have been looking for frameworks to choose from. The only framework that has all the components out of the box for my needs is the Gradio framework. But I do not like how the Gradio apps looks.

I stumbled upon Reflex.dev a couple of days back and I have been looking for methods to create an Image Editor in Reflex. The only thing that comes to my mind is to wrap some React JS library in Reflex. Truthfully speaking, I do not know anything about Javascript and I don't even know what to look for in JavaScript packages.

This is what I need: https://www.gradio.app/docs/imageeditor. Ability to upload an image, draw on top of an image, and erase parts. Undo, Redo changes will be good to have, but not mandatory. 

Can someone please help me in getting started please?

**Comment by Alek99** [comment]
Hello. We don't have a built-in image editor, my suggestion would be to find a react library you like and join our discord. I can help you wrap it in reflex. This would be a cool thing to incorporate in reflex as a third-party library .

We have a section in our docs dedicated to integrating existing react library in reflex https://reflex.dev/docs/wrapping-react/overview/

**Reply by rjachuthan** [reply]
Yes. I was afraid of this. Anyways, I have started looking into the wrapper. But mainly I am currently focussing on the text side of the project and will keep the image side to the end.

Thank you @Alek99  for the quick response.
